### PR TITLE
Feature/use an orb for circleci

### PR DESCRIPTION
**Pull request by zfhui**
Using a [little Orb](https://circleci.com/orbs/registry/orb/zfhui/ruby-gem) that I created to test this gem against different Ruby versions. 

@c-ameron already helped me creating an `blinkist` namespace on CircleCI's Orb Registery, but it's a bit complicated to publish an Orb there. I can only publish development versions under the `blinkist`, only CircleCI owners are allowed to promote/publish a production version. Keeping my own namespace, so I can keep iterating on this for now.

### CODE REVIEW

**Review by c-ameron — APPROVED** [review]
super cool work Fei!!

File: silent_ping.gemspec [hunk 1]
```diff
@@ -37,7 +37,7 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_development_dependency "bundler", "~> 2.0"
+  spec.add_development_dependency "bundler", ">= 1.16.1"
```
Comment (hosseintoussi) [inline silent_ping.gemspec:40]: :( so sad

**Review by hosseintoussi — APPROVED** [review]
really nice setup - i like that both ruby versions are tested.

File: silent_ping.gemspec (hunk 1 above)
Comment (zfhui) [reply silent_ping.gemspec:40]: Yeah, we still need that to support 2.3.6. 😬

**Pull request merged**